analyses/telomerase-activity-prediction/02-Comparing-Counts-versus-FPKM.R

```diff
@@ -0,0 +1,77 @@
+
+
+
+
+################################################ Comparing Counts versus FPKM (Figure 1) ############################################################################################
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git")) 
+
+A = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAPolya_counts.txt")   ### Variable representing Telomerase activity-prediction for Polya_counts
+				 
+B = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAStranded_counts.txt")  ### Variable representing Telomerase activity-prediction for Stranded_counts
+				 
+C = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAPolya_FPKM.txt")      ### Variable representing Telomerase activity-prediction for PolyA_FPKM data
+				 
+D = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAStranded_FPKM.txt")   ### Variable representing Telomerase activity-prediction for Stranded_FPKM data
+
+PTBA_GE_PolyA_TMScores1 = read.table(A,sep='\t',head=T)
+colnames(PTBA_GE_PolyA_TMScores1)[colnames(PTBA_GE_PolyA_TMScores1)=="NormEXTENDScores"]='NormEXTENDScores_PolyACounts'
+
+PTBA_GE_Standard_TMScores1 = read.table(B,sep='\t',head=T)
+colnames(PTBA_GE_Standard_TMScores1)[colnames(PTBA_GE_Standard_TMScores1)=="NormEXTENDScores"]='NormEXTENDScores_StrandedCounts'
+
+PTBA_GE_PolyA_TMScores2 = read.table(C,sep='\t',head=T)
+colnames(PTBA_GE_PolyA_TMScores2)[colnames(PTBA_GE_PolyA_TMScores2)=="NormEXTENDScores"]='NormEXTENDScores_PolyA_FPKM'
+
+PTBA_GE_Standard_TMScores2 = read.table(D,sep='\t',head=T)
+colnames(PTBA_GE_Standard_TMScores2)[colnames(PTBA_GE_Standard_TMScores2)=="NormEXTENDScores"]='NormEXTENDScores_Stranded_FPKM'
+
+PTBA_GE_PolyA_TMScores = merge(PTBA_GE_PolyA_TMScores1,PTBA_GE_PolyA_TMScores2,by='SampleID')
+PTBA_GE_Standard_TMScores = merge(PTBA_GE_Standard_TMScores1,PTBA_GE_Standard_TMScores2,by='SampleID')
+
+
+pdf('plots/PTBA_GE_Score_AllScatter.pdf')
+
+
+P1 = ggscatter(PTBA_GE_PolyA_TMScores, x = "NormEXTENDScores_PolyACounts", y = "NormEXTENDScores_PolyA_FPKM",color = "red",size = 2,
+add = "reg.line",  # Add regressin line
+add.params = list(color = "black", fill = "grey",size=0.5), # Customize reg. line
+conf.int = TRUE # Add confidence interval
+)+ stat_cor(method = "spearman",size=2)
+
+		   
+P2 = ggscatter(PTBA_GE_Standard_TMScores, x = "NormEXTENDScores_StrandedCounts", y = "NormEXTENDScores_Stranded_FPKM",color = "red",size = 1.3,
+add = "reg.line",  # Add regressin line
+add.params = list(color = "black", fill = "grey",size=0.5), # Customize reg. line
+conf.int = TRUE # Add confidence interval
+)+ stat_cor(method = "spearman",size=2)
+		   
+grid.newpage()
+# Create layout : nrow = 2, ncol =2
+pushViewport(viewport(layout = grid.layout(nrow = 2, ncol = 2)))
```

Comment:
Your output is only 2 plots, but the pane is for 2x2, leaving a lot of empty space in the resulting PDF. To fix this, you should make `nrow=1`, and you may (will?) need to fiddle with the PDF dimensions in your call to `pdf()` on line 47 by adding width and/or height arguments.